web/packages/teleport/src/Support/Support.tsx

```diff
@@ -194,16 +192,33 @@ const getDocUrls = (version = '', isEnterprise: boolean) => {
   };
 };
 
-const getDownloadLink = (isCloud: boolean, isEnterprise: boolean) => {
+const DownloadLink = ({
+  isCloud,
+  isEnterprise,
+}: {
+  isCloud: boolean;
+  isEnterprise: boolean;
+}) => {
   if (isCloud) {
-    return cfg.routes.downloadCenter;
+    return (
+      <StyledSupportLink as={Link} to={cfg.routes.downloadCenter}>
```

Comment:
Why do we use `StyledSupportLink` for cloud and `SupportLink` for enterprise?

Reply:
`SupportLink` is just a small wrap around `StyledSupportLink`:

```
const SupportLink = ({ title = '', url = '' }) => (
  <StyledSupportLink href={url} target="_blank">
    {title}
  </StyledSupportLink>
);
```

Reply:
So `SupportLink` is in fact styled. I find this a bit confusing.

Intuitively, if you have a `Foo` and a `StyledFoo`, I'm going to assume that `Foo` is equivalent to `UnstyledFoo`..

Reply:
Yeah, that makes sense to me.

I think `SupportLink` is kind of unnecessary, each link can set its target. 

Or maybe we rename it to `ExternalSupportLink`, so it isn't as confusing between `SupportLink` and `StyledSupportLink`. Thoughts?

Reply:
Either of those sound fine to me. No preference.

Reply:
I went with the ExternalSupportLink route, thanks!